docker-compose.yml

```diff
@@ -156,11 +156,17 @@ services:
     image: eyereasoner/reasoning-service:increased-stack-limit
     volumes:
       - ./config/reasoner:/config
+  privacy:
+    image: lblod/wop-privacy-centric-service:latest
```

Comment:
Could we put a fixed version here instead of latest ? For a little more stability on the prod releases :)